### DISCUSSION TITLE
Throwable components no longer support holdpos="0,0"

### DISCUSSION THREAD

**Discussion by UnhappyWithMyUsername** [FakeFishGames/Barotrauma#13923] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

With the "Blood in the Water" update, throwable components no longer allow to hold an item loosely in your hands and let it swing around with your arms when walking when holpos is configured to be "0,0".
Instead the item will now always be positioned weirdly in front of the character as seen here:
![image](https://github.com/FakeFishGames/Barotrauma/assets/74098498/7da59414-f3c6-47ac-a7f2-96b9caf10f2d)
![image](https://github.com/FakeFishGames/Barotrauma/assets/74098498/cf651e0c-217d-46a9-aa93-d90710dff8e1)

This wasn't an issue before the "Blood in the Water" update.
Was this change intentional? Was there another attribute added to achieve the previous behavior so these items are not automatically positioned?

### Reproduction steps

1. Create an item with a throwable component
2. Spawn the item ingame. Character will hold it in front of them instead of just holding it in their hands with arms just hanging down.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by itchyOwl** [comment]
Not intentional, created a ticket: #13934

**Comment by itchyOwl** [comment]
Fixed in FakeFishGames/Barotrauma-development#5280.
